File: lv0/taller/taller.php
<?php

function solution($list, $height){
    $count = 0;
    foreach($list as $item){
        if ($item > $height){
            $count += 1;
        }
    }
    return $count;
}

$params = [
    [[149, 180, 192, 170], 167, 3],
    [[180, 120, 140], 190, 0],
];

foreach($params as $param){
    $result = solution($param[0], $param[1]);
    echo $result.PHP_EOL;
}

File: lv0/taller/taller.py

def solution(array, height):
    answer = 0
    array.sort()
    for friend_height in array:
        if friend_height > height:
            answer += 1
        elif friend_height == height:
            break
    return answer


params = [
[[149, 180, 192, 170], 167, 3],
[[180, 120, 140], 190, 0],
]

for param in params:
    result = solution(param[0], param[1])
    print(result)
